components/radio/src/__snapshots__/test.tsx.snap

```diff
@@ -268,12 +271,15 @@ exports[`Radio disabled: disabled 1`] = `
 <Radio
   disabled={true}
   inline={false}
+  size="medium"
 >
   <src__Label
     inline={false}
+    size="medium"
   >
     <label
       className="c0"
+      size="medium"
```

Comment:
do we want the size prop to be sent to the label element?

Reply:
@penx the jest snapshots differ from what we have printed to screen. The real output is as:

```
<label class="sc-gsBqHK dEjzGa"><input type="radio" class="sc-aKXVD eEwhHG"><span class="sc-kmATbt cLEvoP">Small radio</span></label>